documentation/src/main/markdown/runningdetect/statelessscan.md

```diff
@@ -8,37 +8,33 @@ Enable this feature by adding [--detect.blackduck.scan.mode=STATELESS](../proper
-    * The currently supported tools are: DETECTOR, BAZEL, SIGNATURE_SCAN, DOCKER, BINARY_SCAN, and CONTAINER_SCAN.
-    * All Stateless Scans will not persist on Black Duck. 
-    * All other tools are disabled when running in Stateless Scan mode.
- * Stateless Scan requires Black Duck policies. 
-    * Stateless Scan only reports components that violate policies. 
-    * If no policies are violated or there are no defined policies, then no components are returned.
- * Stateless Scan does not support ```detect.policy.check.fail.on.severities```
-    * [solution_name] will fail with FAILURE_POLICY_VIOLATION if any component violates Black Duck polices with a CRITICAL or BLOCKER severity. 
-    * See the Black Duck documentation for a list of policy conditions that are supported by Stateless Scan. 
- * Stateless Scan does not support ```detect.policy.check.fail.on.names```
- * Stateless Scan cannot create a Risk or Notices report.
- * Stateless Scan will not create a Project or Version on Black Duck.
- * Stateless Scan when running SIGNATURE_SCAN, BINARY_SCAN, or CONTAINER_SCAN requires communication with Black Duck.
- * Stateless Scan and non-persistent SIGNATURE_SCAN, BINARY_SCAN, or CONTAINER_SCAN
-    * To perform a non-persistent scan in Stateless mode, SIGNATURE_SCAN, BINARY_SCAN, or CONTAINER_SCAN must be included within --detect.tools.
-    * Permitted tools omitted from the detect.tools list will not be run.
+ * Must have Match as a Service (MaaS) enabled within [blackduck_product_name], a feature available as of the [blackduck_product_name] 2022.10.0 release.
+ * Stateless Scan requires [blackduck_product_name] policies retrieved via communication with [blackduck_product_name]. 
 
 ### Signature Scan Requirements
  * Must be running [blackduck_product_name] 2022.10.0 or greater using the hosted KB.
  
-### Binary and Container Scan Requirements
+### Binary Scan Requirements
  * Must be running [blackduck_product_name] 2023.4.0 or greater using the hosted KB.
- * It is necessary to have [solution_name] and [blackduck_product_name] running in the hosted SCAaaS environment to perform these scans. 
- * To run binary or container Stateless Scan a Black Duck Binary Analysis (BDBA) license is required.
+ * It is necessary to have [solution_name] and [blackduck_product_name] running in the hosted environment to perform these scans. 
+ * To run binary scan a Black Duck Binary Analysis (BDBA) license is required.
+ 
+ ### Container Scan Requirements
+ * Must be running [blackduck_product_name] 2023.10.0 or greater to execute container scans that provide container layer findings.
+ * It is necessary to have [solution_name] and [blackduck_product_name] running in the hosted environment to perform these scans. 
+ * To run a Container Stateless Scan your [blackduck_product_name] server must have [blackduck_product_name] Binary Analysis (BDBA) or [blackduck_product_name] Secure Container (BDSC) licensed and enabled.
```

Comment:
Only Secure Container is needed

Reply:
Weird.... could have sworn I made these changes yesterday.
Update committed.